### PR TITLE
made miniature for setup wizard optional

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java [hunk 1]
```diff
@@ -47,12 +48,12 @@
 
     /**
      * Create the setupUi
-     *
-     * @param title title of the setupUi
+     *  @param title title of the setupUi
+     * @param miniature
      * @param userHome
      */
     public UiSetupWizardImplementation(String title,
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: Additional JavaDoc for the remaining parameters, and maybe the class as a whole.

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardFactory.java [hunk 2]
```diff
@@ -23,6 +23,7 @@
 import org.springframework.beans.factory.annotation.Value;
 
 import java.net.URI;
+import java.util.Optional;
 
 public class UiSetupWizardFactory {
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardFactory.java]: JavaDoc

Comment (plata) [reply phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardFactory.java]: Don't know a useful comment which I could add to the class but commented the methods.

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java [hunk 3]
```diff
@@ -46,13 +47,17 @@
...
-     * Create the setupUi
-     *
-     * @param title title of the setupUi
+     * constructor
+     * @param title wizard title
+     * @param miniature miniature for the setup wizard (usually miniature of the application which shall be installed)
+     * @param messageSender
+     * @param setupUiFactory
      * @param userHome
+     * @param applicationUserRoot
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: I'm especially interested in a description of the params that have no description yet^^
For example: what exactly is the `applicatonUserRoot`?

Comment (plata) [reply phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: I didn't know by heart so I didn't write anything. Also, I didn't change this code at all.

**Pull request merged**